CODE_OF_CONDUCT.md

```diff
@@ -50,7 +50,7 @@ decisions when appropriate.
 
 ## Scope
 
-This Code of Conduct applies within all community spaces, and also applies when
+This Code of Conduct applies within all community spaces and also applies when
```

Comment:
issue: placing a comma between two independent clauses is okay, no change needed